### PR TITLE
Adding ConsensusTestShorterLookback and TestAssetCreateWaitBalLookbackDelete

### PR DESCRIPTION
**Pull request by algonautshant**
## Summary
Added ConsensusTestShorterLookback to shorten the lookback rounds from 320 to 32 to test post lookback checks faster.
Added test to create asset, wait  bal lookback rounds,  check asset and delete it
Refactored TestAssetCreateWaitBalLookbackDelete to reuse the subroutines in both tests.

### CODE REVIEW

File: test/e2e-go/features/transactions/asset_test.go [hunk 1]
```diff
@@ -812,37 +829,120 @@ func TestAssetCreateWaitRestartDelete(t *testing.T) {
...
+}
 
-	_, curRound = fixture.GetBalanceAndRound(account0)
-	confirmed = fixture.WaitForAllTxnsToConfirm(curRound+20, txids)
-	a.True(confirmed, "destroying assets")
+
+/** Helper functions **/
+
+// Setup the test and the network 
+func setupTestAndNetwork(t *testing.T, proto string)(
```
Comment (tsachiherman) [inline test/e2e-go/features/transactions/asset_test.go]: rename proto -> networkTemplate

File: test/e2e-go/features/transactions/asset_test.go [hunk 2]
```diff
@@ -812,37 +829,120 @@ func TestAssetCreateWaitRestartDelete(t *testing.T) {
...
+	Assertions *require.Assertions, Fixture *fixtures.RestClientFixture,  Client *libgoal.Client, Account0 string) {
+
+	t.Parallel()
+	asser := require.New(t)
+	if 0 == len(proto){
+		// If the  protocol is not specified, used the default one
+		proto = "TwoNodes50EachFuture.json"
```
Comment (tsachiherman) [inline test/e2e-go/features/transactions/asset_test.go]: don't use "future" anymore. we're already in the future

File: config/config.go [hunk 3]
```diff
@@ -492,6 +492,20 @@ func initConsensusTestProtocols() {
...
+	// Will result in MaxBalLookback = 32 
+	// Used to run tests faster where past MaxBalLookback values are checked
+	testShorterLookback := Consensus[protocol.ConsensusFuture]
+	testShorterLookback.ApprovedUpgrades = map[protocol.ConsensusVersion]bool{}
+
+	// MaxBalLookback  =  2 x SeedRefreshInterval x SeedLookback
+	// ref. https://github.com/algorandfoundation/specs/blob/master/dev/abft.md
+	testShorterLookback.SeedLookback = 2
+	testShorterLookback.SeedRefreshInterval = 8
+	testShorterLookback.MaxBalLookback = // 32
```
Comment (tsachiherman) [inline config/config.go]: make into one line, add `// 32 ` as a comment at the end of the line.

File: test/e2e-go/features/transactions/asset_test.go (hunk 1 above)
Comment (algonautshant) [reply test/e2e-go/features/transactions/asset_test.go]: changed

File: test/e2e-go/features/transactions/asset_test.go (hunk 2 above)
Comment (algonautshant) [reply test/e2e-go/features/transactions/asset_test.go]: This is already in the test configuration. We need to remove "future" from many places to make it consistent.

**Pull request merged**